add_gist.rb

```diff
@@ -0,0 +1,64 @@
+require 'rubygems'
+require 'net/http'
+require 'uri'
+require 'json'
+require 'OpenSSL'
+require 'find'
+require 'dotenv/load'
+
+module AddGist
+  GISTS_URL = 'https://api.github.com/gists'.freeze
+  SUCCESS_CODE = '201'.freeze
+
+  private_class_method def self.read_files(pwd = Dir.pwd)
+    files = {}
+    Find.find(pwd) do |path|
+      files.merge!(File.basename(path) => { 'content': File.open(path, 'r+').read }) unless File.directory?(path)
+    end
+    files
+  rescue Errno::ENOENT => e
+    puts e.message
+    exit
```

Comment:
do we need this `exit`? or can we consider the nil response in the `upload_files` method?

Reply:
I didn't consider the nil response in the ```upload_files``` method and understood we wanted to stop the program execution when the path was not found, that's why I used ```exit```.